vassal-doc/src/main/readme-referencemanual/ReferenceManual/Replace.adoc

```diff
@@ -6,45 +6,48 @@
 
-*Command:*  The right-click context menu text for this trait.
+*Menu command:*::  The right-click context menu text for this trait.
 If left blank, no context menu item will appear, but the trait can still be activated with the Key Command.
 
-*Key Command:*  The <<NamedKeyCommand.adoc#top,Keystroke or Named Command>> to initiate this trait.
+*Key command:*::  The <<NamedKeyCommand.adoc#top,Keystroke or Named Command>> to initiate this trait.
 
-*Define Replacement:*  Click the _Define Replacement_ button to open a Piece Definer to create a definition for the new piece from scratch (or to edit the definition you have already created).
-
-*Select:*  Click the _Select_ button to use the definition from an existing piece in the module.
+*Marker definition:*::  Click the *_Define Replacement_* button to open a Piece Definer to create a definition for the new piece from scratch (or to edit the definition you have already created).
++
+Click the *_Select_* button to use the definition from an existing piece in the module.
 This acts as a pointer to an existing definition: if the original piece is changed, the piece created by the _Replace with Other_ trait will also change.
 
-*Horizontal offset:*  The center of the replacement piece will be placed this many pixels to the right of the center of the original piece.
+*Horizontal offset:*::  The center of the replacement piece will be placed this many pixels to the right of the center of the original piece.
 
-*Vertical offset:*  The center of the replacement piece will be placed this many pixels above the center of the original piece.
+*Vertical offset:*::  The center of the replacement piece will be placed this many pixels above the center of the original piece.
 
-*Match Current State:*  If selected, VASSAL will attempt to put the replacement piece in the same state as the original piece.
+*Match current state:*::  If selected, VASSAL will attempt to put the replacement piece in the same state as the original piece.
 Layers will be set to the same level, labels will be given the same value, rotation angles will match, etc.
 The state of a particular trait will carry over only if it has an exact match in the replacement, i.e.
 the properties settings of that trait are the same in both the original and replacement piece.
 
-*Only match states above this trait:*  If selected, VASSAL will only replace states in traits that appear above this one in the list traits in the <<GamePiece.adoc#top,game piece editor>>. For example, the state of a <<PropertyMarker.adoc#top,Marker>> that's above this one _will_ change if the state in the new game piece.
+*Only match states above this trait:*::  If selected, VASSAL will only replace states in traits that appear above this one in the list traits in the <<GamePiece.adoc#top,game piece editor>>. For example, the state of a <<PropertyMarker.adoc#top,Marker>> that's above this one _will_ change if the state in the new game piece.
```

Comment:
Missing "of" between "list" and "traits".  Also, the example seems to be an incomplete sentence, and I'm not entirely sure what it's trying to say or how to fix it....